### PR TITLE
chore: remove old iOS/Xcode related test code and errors

### PR DESCRIPTION
**Pull request by KazuCocoa**
Found Xcode 11/12/old iOS related code. This PR just removes them.

### CODE REVIEW

File: WebDriverAgentTests/IntegrationTests/FBElementVisibilityTests.m [hunk 1]
```diff
@@ -63,11 +63,6 @@ - (void)disabled_testIconsFromSearchDashboard
 
 - (void)testTableViewCells
 {
-  if (SYSTEM_VERSION_GREATER_THAN(@"12.0")) {
-    // The test is flacky on iOS 12+ in Travis env
-    return;
-  }
```
Comment (KazuCocoa) [inline WebDriverAgentTests/IntegrationTests/FBElementVisibilityTests.m:69]: this test worked on my local. lets bring them back

File: WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m [hunk 2]
```diff
@@ -298,18 +280,13 @@ - (void)testDescendantsWithClassChainWithIndex
...
-                                                     shouldReturnAfterFirstMatch:NO];
-    if (matchingSnapshots.count == 0) {
-      // iPad
-      queryString = @"XCUIElementTypeWindow/*/*/*/*/*[2]/*/*/XCUIElementTypeButton";
-      matchingSnapshots = [self.testedApplication fb_descendantsMatchingClassChain:queryString
-                                                       shouldReturnAfterFirstMatch:NO];
-    }
-  } else {
+  // iPhone
+  queryString = @"XCUIElementTypeWindow/*/*/*/*[2]/*/*/XCUIElementTypeButton";
```
Comment (mykola-mokhnach) [inline WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m]: do we need now the above expression that initializes queryString?

**Review by mykola-mokhnach — APPROVED** [review]
I love such PRs

File: WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m (hunk 2 above)
Comment (KazuCocoa) [reply WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m]: oh, yea. will modify here

**Pull request merged**